src/components/schedule-page/table/schedule/schedule-parts/base-cell/base-cell.component.tsx

```diff
@@ -52,21 +58,30 @@ export function BaseCellComponent({
     onValueChange && onValueChange(newValue);
   }
 
+  function getId(): string {
+    if (verboseDate && monthNumber) {
+      if (verboseDate.month !== TranslationHelper.englishMonths[monthNumber]) {
+        return "otherMonth";
+      }
+      if (
+        verboseDate.isPublicHoliday ||
+        verboseDate.dayOfWeek === WeekDay.SA ||
+        verboseDate.dayOfWeek === WeekDay.SU
+      ) {
+        return "weekend";
+      }
+    }
+    return "thisMonth";
+  }
   //  #region view
   return (
     <td
-      className="cell"
+      className="mainCell"
+      id={getId()}
       onClick={(): void => {
         !isBlocked && onClick && onClick();
       }}
       onContextMenu={handleContextMenu}
-      style={{
-        color: style.textColor.toString(),
-        backgroundColor:
-          isSelected || isPointerOn
-            ? ColorHelper.getHighlightColor().toString()
```

Comment:
Ta linijka gwarantowała to, że przy wyborze zakresu komórek, one były podświetlane, a teraz nie są. Ponadto, `ColorHelper` był potrzebny do tego, żeby mieć takie same kolory w wyeksportowanym excelu i na grafiku. Mi się wydaje, że takie use case jest dość sensowny i ja bym go zostawił, tylko zmieniłbym zwracane przez niego kolory: np. dałbym opacity na 0.1 - wtedy to będzie zdecydowanie bardziej pasować to tabelki niż poprzednia implementacja

Reply:
@kaplonpaulina i co z tą funkcjonalnością? Czy będziesz ją przywracać w tym PR czy wygląda to słabo?